src/Swift.Runtime/src/Swift/SwiftString.cs

```diff
@@ -146,14 +160,19 @@ public override string ToString()
         if (length <= 0)
             return string.Empty;
 
-        var contiguousArray = PInvoke_GetUtf8ContiguousArray(_payload);
+        bool _success = false;
+        _payload.DangerousAddRef(ref _success);
```

Comment:
This should be done when no exceptions are possible or prior to a `try` block.

Can `PInvoke_GetUtf8ContiguousArray` throw during the marshal?

Reply:
Yes, good point. Added try/finally block.